docs/2_3_fmu_distribution.adoc

```diff
@@ -43,6 +43,9 @@ resources                     // resources needed by the FMU (optional)
    // In order for the FMU to access these resource files, the resource directory
    // must be available in unzipped form and the absolute path to this directory
    // must be reported via argument "fmuResourceLocation" via fmi3Instantiate.
+extra                         // Additional meta-data of the FMU (optional)
```

Comment:
If the folder only contains metadata, shouldn't it be `meta` or `metadata`?

Reply:
But then for eFMI from the EMPHYSIS project we would then need on the same level an `eFMU` folder.

Reply:
I was just wondering why "extra".

Reply:
The actual reason: Because SSP already uses extra. And SSP didn't use meta-data because it is not really always clear what is data and what is meta-data. So I think I'll try to reword the comment to not (only) talk about meta-data.

There are probably better names, but it would be nice to use the same as SSP.